FalmouthFalcons/iteration-2
# adam-customer-select method and conditions

Description:
_Conditional logic is working on the CLI via main_menu_controller for active customer, must be selected before option 3 may be accessed from main menu. 
_payment type  added to customer confirmation message.

Number of Fixes:
-Finished active_customer functionalitty from #2 .

Related Ticket(s):
#2 #3 

Problem to Solve:
- conditional active customer 

Proposed Changes:
-changes to the main_menu_controller, payment_type_controller, and customers_controller 

Steps to Test Solution
1) git pull origin master 
2)git fetch 
3) git checkout adam-customer-select

Documentation
[N ] There is new documentation in this pull request that must be reviewed..
[Y ] I added documentation for any new classes/methods

Deploy Notes:
- Notes regarding deployment the contained body of work.
- These should note any db migrations, etc.
